media/css/mozorg/home/home-en.scss

```diff
@@ -22,60 +21,26 @@ $image-path: '/media/protocol/img';
     @include visually-hidden;
 }
 
-//* -------------------------------------------------------------------------- */
-// Primary CTA (page top)
+/* -------------------------------------------------------------------------- */
+// Split container
 
-.privacy-promise-hero.mzp-c-hero.mzp-t-dark {
+.mzp-c-split.mzp-t-dark {
     background-color: $color-ink-80;
-    overflow: hidden;
-    padding-bottom: 0;
-    text-align: left;
 
-    .mzp-c-hero-title {
-        @include font-firefox;
-        @include font-size(32px);
+    .mzp-c-split-media {
+        max-width: 400px;
     }
 
-    .mzp-c-hero-body {
-        max-width: none;
-    }
-
-    .privacy-promise-hero-desc {
-        @include text-body-lg;
-        margin-bottom: 0;
-    }
-
-    .mzp-c-hero-image {
-        display: none;
-    }
-
-    @media #{$mq-md} {
-        .privacy-promise-hero-desc {
-            margin-bottom: $spacing-lg;
-        }
-
-        .mzp-c-hero-image {
-            display: block;
-            height: 100%;
-            margin: 0;
-
-            img {
-                @include bidi(((left, $spacing-xl, right, auto),));
-            }
-        }
-    }
-
-    @media #{$mq-lg} {
-        padding-bottom: $spacing-lg;
-
-        .mzp-c-hero-title {
-            @include font-size(48px);
+    .mzp-c-split-container {
+        padding-top: $spacing-lg;
+        padding-bottom: 0;
+        @media #{$mq-lg} {
+            padding-bottom: 0;
+            box-sizing: content-box;
```

Comment:
Same comment here for overriding the vendor prefix values as well as the default `box-sizing` property.